resource/templates/retrain-recipe-template.html

```diff
@@ -120,6 +120,11 @@ <h2 class="settings-section-title">Optimization</h2>
             </div>
             <div class="controls">
                 <button class="btn btn-default" ng-click="addCustomParam('model_custom_params_opti')">Add</button>
+                <span class="help-inline">See available parameters here:
+                    <a href="https://keras.io/api/optimizers/#available-optimizers" target="_blank">
+                        https://keras.io/api/optimizers/#available-optimizers
+                    </a>
+                </span>
             </div>
         </div>
         <h2 class="settings-section-title">Training</h2>
```

Comment:
Maybe you could put the Tensorboard button before the Data Augmentation button; this way you could replace the button for custom parameters now called "Data augment. custom params" simply by "Custom parameters", since it won't be ambiguous anymore ? Or, if you don't want to, you could call it "Data augmentation parameters" directly.

Reply:
You are definitely right, it can be improved! I added sections in afcd3b1 to make it more clear, what do you think? Isn't it too much? I'd like to keep tensorboard at the end as it is a specific feature so it should not be mixed up with the training parameters

Reply:
This is definitely more clear ! I just have a doubt on having sections for only one parameter. I like the idea to have a section for data augmentation, but the tensorboard one is a little bit disturbing to me. Can't you put them together in an "advanced options" section for example ? Or go back to the previous version but with the added arrows.  Here are two suggestions, be free to do something else
<img width="579" alt="Capture d’écran 2021-03-26 à 11 21 00" src="https://user-images.githubusercontent.com/77485161/112617607-769f0800-8e25-11eb-9388-06e1b04eb00e.png">

Reply:
It is better with the advanced section, thanks! Fixed in 2dd3103